### PR TITLE
Replace dataproc-initialization-actions with <my-bucket> in README

### PR DESCRIPTION
**Pull request by functicons**

### CODE REVIEW

File: beam/README.md [hunk 1]
```diff
@@ -111,9 +111,9 @@ shown later).
 
 ```bash
 CLUSTER_NAME="$1
-INIT_ACTIONS="gs://dataproc-initialization-actions/docker/docker.sh"
-INIT_ACTIONS+=",gs://dataproc-initialization-actions/flink/flink.sh"
-INIT_ACTIONS+=",gs://dataproc-initialization-actions/beam/beam.sh"
+INIT_ACTIONS="gs://<my-bucket>/docker/docker.sh"
```
Comment (medb) [inline beam/README.md]: Replace with `$BUCKET` variable so it will be copy-pastable?

**Comment by findepi** [comment]
@functicons in fact, why?

**Comment by functicons** [comment]
> @functicons in fact, why?

We want to discourage users referencing gs://dataproc-initialization-actions directly, because it is changing, we have broken users in the past.

**Pull request merged**